lib/params.js

```diff
@@ -21,9 +21,9 @@ function hasScssInWorkingDir (wd) {
 function getIndentationFromEditorConfig (from, hasScss) {
   var config
   if (hasScss) {
-    config = editorconfig.parseSync(path.resolve(process.cwd(), '*.scss'))
+    config = editorconfig.parseSync(path.resolve(from, '*.scss'))
```

Comment:
I think we should use real file path to find `.editorconfig`. I will commit new code.